server/index.js

```diff
@@ -33,6 +34,58 @@ app.get('/additional/:id', (req, res) => {
   }
 });
 
+app.post('/additional', (req, res) => {
+  const newItem = new Item({
+    sellerName: req.body.sellerName,
+    sellerCountry: req.body.sellerCountry,
+    sellerTotalSales: req.body.sellerTotalSales,
+    sellerJoinDate: req.body.sellerJoinDate,
+    sellerPicture: req.body.sellerPicture,
+    sellerStarRating: req.body.sellerStarRating,
+    sellerReviewCount: req.body.sellerReviewCount,
+    itemId: req.body.itemId,
+    itemName: req.body.itemName,
+    itemPrice: req.body.itemPrice,
+    itemPicture: req.body.itemPicture,
+    itemFreeShipping: req.body.itemFreeShipping,
+  });
+
+  newItem.save()
+    .then(() => res.json('Item saved'))
+    .catch((err) => res.status(400).json({ Error: err }));
+});
+
+app.put('/additional/:id', (req, res) => {
+  const { id } = req.params;
+
+  const item = {
+    sellerName: req.body.sellerName,
+    sellerCountry: req.body.sellerCountry,
```

Comment:
Let me share my thoughts:
Here you have references to different countries (some countable number ~200 countries in the world). So when You will move to SQL db, you can create a reference to a separate table with countries. So that when your sql-DB will store 10M numbers from 1 to 200 and a separate matrix 2*200 (approx.) Instead of storing 10M strings. But MAY be your will lose in speed because you will need to do joins.

Reply:
Yes, I'm not looking forward to having a separate table and doing joins and losing speed :(